app.js

```diff
@@ -0,0 +1,92 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Store {
+  static getBooks() {
+    let books;
```

Comment:
- [ ] [OPTIONAL] In my opinion, your project is working well and ready to approve, but the way you did this milestone is the requirements required in the feature milestone😄  But you have implemented all the required functionalities, I applaud you, keep doing 👍🏼